### PR TITLE
Auto git tag

### PR DESCRIPTION
**Pull request by hensha256**
Given we are unreliable with git tags, I wanted to automate it

### CODE REVIEW

**Review by marktoda — COMMENTED** [review]
hahahah nice - one problem is if you update with `npm version <patch|minor|major>` the tag will already be created so the script will fail?

**Comment by hensha256** [comment]
oh yeah i always do it manually... can you think of a way of changing this so it works with that too?

File: package.json [hunk 1]
```diff
@@ -68,6 +68,7 @@
     "symlink": "ln -s ./lib/solmate ./solmate && ln -s ./lib/permit2 ./permit2",
     "prettier:fix": "prettier --write '**/*.ts' && prettier --write '**/*.json'",
     "lint:fix": "yarn prettier:fix && forge fmt",
-    "prettier": "prettier --check '**/*.ts' && forge fmt --check"
+    "prettier": "prettier --check '**/*.ts' && forge fmt --check",
+    "postpublish": "PACKAGE_VERSION=$(cat package.json | grep \\\"version\\\" | head -1 | awk -F: '{ print $2 }' | sed 's/[\",]//g' | tr -d '[[:space:]]') && git tag v$PACKAGE_VERSION && git push --tags"
```
[the remark this answers is not in the document]
Comment (marktoda) [reply package.json:72]: oh i guess this list isnt sorted at all

[the remark this answers is not in the document]
Comment (hensha256) [reply package.json:72]: yeah this whole file is out of whack i'll reorder them all

**Comment by hensha256** [comment]
okay killing it, mark and i chatted on slack

**Pull request closed without merging**